packages/design-system/src/components/Table/Table.test.tsx

```diff
@@ -40,6 +40,12 @@ describe('Table', function () {
     expect(screen.getByRole('table')).toHaveClass('ds-c-table--striped');
   });
 
+  it('accepts custom id', () => {
+    makeTable({ id: 'static-id', scrollable: true });
```

Comment:
`id` only appears when table is scrollable

Reply:
oh interesting